### DISCUSSION TITLE
Formik-Stepper @2.0.0

### DISCUSSION THREAD

**Discussion by Riyad-Arafat** [Riyad-Arafat/formik-stepper#9] [General]
This is the second version of that package in which we have added some new features and improved performance and reduced some of the props in some components in order to help ease of use. And we have added some new input fields like [SelectField, RadioField]. I hope you like it :)

<hr /><em>This discussion was created from the release <a href='https://github.com/riyadelberkawy/formik-stepper/releases/tag/2.0.0'>Formik-Stepper @2.0.0</a>.</em>

**Comment by rockingrohit9639** [comment]
Hey @riyadelberkawy 
How can I use different validation schema for different steps?

**Reply by Riyad-Arafat** [reply]
Hey @rockingrohit9639 

You write one validation system for all fields and when submitting the current step, the formik-stepper validates all the fields in the current step and cannot move to the next one before validating the current step.

You could take a look at the demo example
https://formik-stepper.web.app/

**Reply by rockingrohit9639** [reply]
So, is it necessary to have initial values to make it work? @riyadelberkawy

**Reply by Riyad-Arafat** [reply]
No, it's not
@rockingrohit9639

**Reply by rockingrohit9639** [reply]
> No, it's not @rockingrohit9639

It is giving me the following error if I have included the field of next step in the validationSchema

<img width="425" alt="image" src="https://user-images.githubusercontent.com/40729749/194855233-4fcd2a63-5343-4859-adab-009d3825740b.png">

If I remove the field from validation schema then it works
